Apps/Src/main.c

```diff
@@ -44,6 +44,8 @@ int main(){
 
 	BSP_WDTimer_Start();
 
+	uint32_t heartcount = 0;
```

Comment:
When I compile I get " warning: unused variable ‘heartcount’ [-Wunused-variable]" because heartcount is declared as local to main().